src/App.tsx

```diff
@@ -133,9 +141,10 @@ function App() {
             </select>
           </div>
```

Comment:
Also please check why we dont have script file name input 
![Screenshot from 2022-05-23 15-11-24](https://user-images.githubusercontent.com/25555466/169806904-124b3ec7-42c4-4aed-adc2-fa6894a771fd.png)

Reply:
the script file name is hard coded into App.tsx component

Reply:
![Screenshot from 2022-05-23 15-32-09](https://user-images.githubusercontent.com/25555466/169810243-8a6cd09f-5966-434c-b432-e9025d723934.png)
Yeah, but when I run the project locally the input is absent